CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+namespace CanHazFunny;
+
+public class Jester
+{
+	public JokeService _jokeService { get; set; }
+	public JokeOutput _jokeOutput { get; set; }
+
+	public Jester (JokeService jokeService, JokeOutput jokeOutput)
+	{
+		if(jokeService is null)
+		{
+			throw new ArgumentNullException(nameof(_jokeService));
+		}
+		if (jokeOutput is null)
+		{
+            throw new ArgumentNullException(nameof(_jokeOutput));
+        }
+		_jokeService = jokeService;
+		_jokeOutput = jokeOutput;
+	}
```

Comment:
```suggestion
	{
	    _jokeService = jokeService ?? throw new ArgumentNullException(nameof(jokeService));
            _jokeOutput = jokeOutput ?? throw new ArgumentNullException(nameof(jokeOutput));
	}
```
You can do the null checking inline, also, make sure you are putting the argument inside the null exception, not the thing you are setting to the argument.